src/Components/Reviews.jsx

```diff
@@ -24,12 +27,16 @@ const Reviews = () => {
       .catch((err) => {
         console.dir(err);
       });
-  }, [category]);
+  }, [category, orderBy, sortBy]);
 
   return loading ? (
     <div className="loader">...Loading</div>
   ) : isNaN(Number(category)) ? (
     <main className="Main">
+      <SortReviews
+        setSortBy={[sortBy, setSortBy]}
+        setOrderBy={[orderBy, setOrderBy]}
```

Comment:
We could pass the sortBy and setSortBy etc as separate props so that we can easily destructure them in the component. It feels like an anti pattern here to put them in an array, only to immediately remove them from the array in the component